src/vector/platform/ElectronPlatform.tsx

```diff
@@ -339,7 +339,7 @@ export default class ElectronPlatform extends VectorBasePlatform {
     ) {
         // this will get intercepted by electron-main will-navigate
         super.startSingleSignOn(mxClient, loginType, fragmentAfterLogin, idpId);
-        Modal.createTrackedDialog('Electron', 'SSO', InfoDialog, {
+        Modal.createDialog(InfoDialog, {
```

Comment:
Why is this being un-tracked?

Reply:
nevermind, realized when reading the react-sdk side that `createTrackedDialog` was a piwik-specific thing